Ktisis/Locale/i18n/fr_FR.json

```diff
@@ -0,0 +1,407 @@
+{
+	"$meta": {
+		"displayName": "French",
+		"selfName": "French",
+		"__comment": "Utilisation de l'élément null pour signaler 'et al' dans le tableau ci-dessous. Ne faites pas cela pour votre propre localisation.",
```

Comment:
By the way, there was no need to translate `__comment` stuff as this is used for translators and will never be displayed in game.